auth/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "auth"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+mod_use   = "0.2.0"
+thiserror = "1.0.30"
+argon2    = { version = "0.4.0", features = ["std"] }
+serde     = { version = "1.0.136", features = ["derive"] }
+mongodb   = { version = "2.2.0-beta", features = ["bson-uuid-0_8"] }
```

Comment:
sort dependencies

Reply:
and should we lock patch or minor?

Reply:
> and should we lock patch or minor?

This is equivalent to ^ in semver, which allows patch update.

Reply:
> sort dependencies

No. This sorting is intended. Sort according to length from short to long.